tests/codegen/targets/cpp_test.py

```diff
@@ -0,0 +1,196 @@
+import unittest
+from functools import reduce
+from operator import mul
+
+import dace
+from dace import SDFG, Memlet
+from dace.codegen.targets import cpp
+from dace.subsets import Range
+from dace.transformation.dataflow import RedundantArray
+
+
+class ReshapeStrides(unittest.TestCase):
```

Comment:
please use pytest as we use it in the other tests (i.e., as free functions), and place the file in tests/codegen.

Reply:
Done.